packages/core/src/hydration/interfaces.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.dev/license
  */
 
+import {HydrateTrigger, Trigger} from '../defer/interfaces';
```

Comment:
This change should help avoid circular deps:
```suggestion
import type {HydrateTrigger, Trigger} from '../defer/interfaces';
```